### PR TITLE
Add hg_prompt_info

### PR DESCRIPTION
**Pull request by kevinmost**

### CODE REVIEW

File: mira.zsh-theme [hunk 1]
```diff
@@ -10,12 +10,12 @@ else
 fi
 
 local current_dir='%{$terminfo[bold]$fg[blue]%}%~ %{$reset_color%}'
-local git_branch='$(git_prompt_info)'
+local vcs_branch='$(git_prompt_info)$(hg_prompt_info)'
```
Comment (mbStavola) [inline mira.zsh-theme:22]: Where does `hg_prompt_info` come from? Is it native to `oh-my-zsh`? I can't find anything related to this from a cursory Google search.

Comment (kevinmost) [reply mira.zsh-theme:22]: Good catch; turns out this comes from a 3rd-party plugin. I documented that above the `local vcs_branch`.

Comment (mbStavola) [reply mira.zsh-theme:22]: Sounds good!

**Review by mbStavola — APPROVED** [review]
Thanks for the PR!

**Pull request merged**